pkg/apis/service/v1alpha1/types.go

```diff
@@ -43,6 +46,27 @@ type FalcoServiceConfig struct {
 	// Configuration for custom webhook
 	// +optional
 	CustomWebhook *Webhook `json:"webhook,omitempty"`
+
+	// -------------------------------------------------------------------
+	// added due to issue #215
+
+	Rules *Rules `json:"rules,omitempty"`
+
+	Destinations *[]Destination `json:"destinations,omitempty"`
+}
+
+type Destination struct {
+	Name              string  `json:"name,omitempty"`
+	ResourceSecretRef *string `json:"resourceSecretRef,omitempty"`
+}
+
+type Rules struct {
+	StandardRules *[]string     `json:"standard,omitempty"`
+	CustomRules   *[]CustomRule `json:"custom,omitempty"`
```

Comment:
Same here?

Also, the `Rules` suffix could be omitted (struct name already scopes this)

Reply:
same here